dbt-redshift/tests/functional/adapter/test_exception_handling.py

```diff
@@ -32,6 +32,7 @@
 
                       delete from {{target.schema}}.test_table where test_col in (select test_col from trans_temp_table);
                       insert into {{target.schema}}.test_table  (select * from trans_temp_table);
+                      insert into {{target.schema}}.test_table  (select * from trans_temp_table);
```

Comment:
I think this only gets used in the test that we're now skipping. Was this intentionally updated or is it an artifact of troubleshooting?

Reply:
artifact of trouble shooting.... will delete